docs/release-notes/8.13.asciidoc

```diff
@@ -0,0 +1,48 @@
+[[release-notes-header-8.13.0]]
+== 8.13
+
+[discrete]
+[[release-notes-8.13.0]]
+=== 8.13.0
+
+[discrete]
+[[features-8.13.0]]
+==== Features
+
+* Allows you to define an entity's (such as a host's or user's) `Asset criticality`, which can affect risk scores ({pull}176815[#176815], {pull}176294[#176294], {pull}172417[#172417], {pull}176056[#176056]).
+* Information on the Data Quality dashboard now persists in {elastic-sec} rather than disappearing after each session ({pull}175673[#175673], {pull}173185[#173185]).
+* Adds field-by-field diffs to the rules upgrade flyout so you can see what's changed between versions ({pull}174564[#174564]).
+* Adds alert suppression to the Indicator Match rule type ({pull}174241[#174241]).
+* You can add Elastic Defend’s `kill-process` or `suspend-process` response actions to detection rules to automatically terminate or suspend a process on an affected host ({pull}161645[#161645]).
+* You can isolate and release a SentinelOne-protected host from detection alerts and the response console, and view third-party actions in the response actions history log ({pull}173927[#173927], {pull}175810[#175810]).
+* Allows you to enable and disable cloud security Benchmark rules ({pull}174575[#174575]).
+
+
+[discrete]
+[[enhancements-8.13.0]]
+==== Enhancements
+
+* Enables advanced sorting and customization options for the the Findings page's **Vulnerabilities** table ({pull}174413[#174413]).
+* Adds the ability to analyze an event within a specific time range and data view ({pull}176364[#176364]).
+* Enables the newly expanded Host and User details flyouts by default ({pull}175899[#175899]).
+* Improves the header layout in the alert details flyout so basic alert details are better organized ({pull}175075[#175075]).
+* Adds inline actions and a search bar to the left panel in the event analyzer UI and improves formatting issues ({pull}172397[#172397]).
+
+[discrete]
+[[bug-fixes-8.13.0]]
```

Comment:
I think we're missing the following item: https://github.com/elastic/kibana/pull/178537 (merged yesterday)

Reply:
Ya sounds good. Sorry about the late merge, it's been on auto merge since last Wednesday and only unrelated tests were failing

Reply:
@nastasha-solomon @PhilippeOberti actually this backport pr has been failing, and seeing how the last BC is being built sometime today, I think we should delay this until 13.1 at this point.